src/lib/yang/schema.lua

```diff
@@ -884,6 +908,25 @@ function load_schema_by_name(name, revision)
 end
 load_schema_by_name = util.memoize(load_schema_by_name)
 
+function lookup_identity (fqid)
+   local schema_name, id = fqid:match("^([^:]*):(.*)$")
+   local schema, env = load_schema_by_name(schema_name)
+   local id_thunk = env.identities[id]
+   if not id_thunk then
+      error('no identity '..id..' in module '..schema_name)
+   end
+   return id_thunk() -- Force the lazy lookup.
+end
```

Comment:
I don't know why this comment is not marked as outdated. The changes was addressed in the amend patch.